lib/ts/recipe/multifactorauth/api/implementation.ts

```diff
@@ -25,89 +25,74 @@ export default function getAPIInterface(): APIInterface {
 
```

Comment:
no need to call `await session.fetchAndSetClaim(MultiFactorAuthClaim, userContext); // ensure claim value is present` above

Reply:
Done